libs/application/templates/criminal-record/src/lib/CriminalRecordTemplate.ts

```diff
@@ -0,0 +1,141 @@
+        meta: {
+          name: 'Umsókn um sakavottorð',
+          actionCard: {
+            tag: {
+              label: m.actionCardDraft,
+              variant: 'blue',
+            },
+          },
+          progress: 0.25,
+          lifecycle: {
+            shouldBeListed: false,
+            shouldBePruned: true,
+            // Applications that stay in this state for 24 hours will be pruned automatically
+            whenToPrune: 24 * 3600 * 1000,
+          },
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/CriminalRecordForm').then((module) =>
+                  Promise.resolve(module.CriminalRecordForm),
+                ),
+              actions: [
+                {
+                  event: DefaultEvents.PAYMENT,
+                  name: 'Staðfesta',
+                  type: 'primary',
+                },
+              ],
+              write: 'all',
+            },
+          ],
+        },
+        on: {
+          [DefaultEvents.PAYMENT]: {
+            target: States.PAYMENT,
+          },
+          [DefaultEvents.SUBMIT]: { target: States.PAYMENT },
+        },
+      },
+      [States.PAYMENT]: {
+        meta: {
+          name: 'Greiðsla',
+          actionCard: {
+            tag: {
+              label: m.actionCardPayment,
+              variant: 'red',
+            },
+          },
+          progress: 0.8,
+          lifecycle: DefaultStateLifeCycle,
+          onEntry: {
+            apiModuleAction: ApiActions.createCharge,
+          },
+          onExit: {
+            apiModuleAction: ApiActions.submitApplication,
+          },
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/Payment').then((val) => val.Payment),
+              actions: [
+                { event: DefaultEvents.SUBMIT, name: 'Áfram', type: 'primary' },
+              ],
+              write: 'all',
+            },
+          ],
+        },
+        on: {
+          [DefaultEvents.SUBMIT]: { target: States.COMPLETED },
+        },
+      },
+      [States.COMPLETED]: {
+        meta: {
+          name: 'Completed',
+          progress: 1,
+          lifecycle: DefaultStateLifeCycle,
+          actionCard: {
+            tag: {
+              label: m.actionCardDone,
+              variant: 'blueberry',
+            },
+          },
+          onEntry: {
+            apiModuleAction: ApiActions.getCriminalRecord,
+          },
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/Approved').then((val) =>
+                  Promise.resolve(val.Approved),
+                ),
+              read: 'all',
+            },
+          ],
+        },
+        type: 'final' as const,
+      },
+    },
+  },
+  mapUserToRole(): ApplicationRole | undefined {
+    return Roles.APPLICANT
```

Comment:
You only want to return this role if the `nationalId` (passed in as the first argument) is equal to `application.applicant` (application is passed in as the second argument).